polaris.shopify.com/pages/tokens/spacing.tsx

```diff
@@ -6,8 +6,8 @@ import PageMeta from '../../src/components/PageMeta';
 const Components: NextPage = () => {
   return (
     <>
-      <PageMeta title="Spacing tokens" />
-      <TokensPage tokenGroup={'spacing'} />
+      <PageMeta title="Space tokens" />
+      <TokensPage tokenGroup={'space'} />
```

Comment:
I'd prefer to handle this change in https://github.com/Shopify/polaris/pull/9006